src/pages/waiting-room/waiting-room.ts

```diff
@@ -193,7 +194,15 @@ export class WaitingRoomPage extends PracticeableBasePageComponent implements On
       this.deviceAuthenticationProvider.triggerLockScreen()
         .then(() => {
           this.store$.dispatch(new waitingRoomActions.SubmitWaitingRoomInfo());
-          this.navCtrl.push('WaitingRoomToCarPage');
+          this.navController.push(WAITING_ROOM_TO_CAR_PAGE).then(() => {
+            const communicationPage = this.navController.getViews().find(view => view.name === COMMUNICATION_PAGE);
+            if (communicationPage) {
+              this.navController.removeView(communicationPage);
```

Comment:
I thought you could go back to communication page if you wanted to?

Reply:
You can from Waiting Room but not from Waiting Room To Car